.rubocop.yml

```diff
@@ -29,21 +23,9 @@ Layout/IndentHash:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: false
 
-Metrics/ClassLength:
-  Enabled: false
-
-Metrics/LineLength:
```

Comment:
How about specifying what line length we're ok with?  Something like:
```Metrics/LineLength:
  Max: 160```
and then we can see what it catches.

Reply:
I use 120, but let's go with 150 — that seems to be a standard.